### PR TITLE
Shell recipes tweaks

### PR DESCRIPTION
**Pull request by tomaszmrugalski**
Small tweaks in the recipes:

- meteor is now able to use either meteor_demod or meteor-demod (whichever is available in the system)
- recipes now refuse to start if not given 3 parameters
- shellcheck complaints fixed
- updated travis CI (added shellcheck for shell scripts, so far only recipes are tested, more to come)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,8 +32,9 @@ before_script:
   - mkdir -p ~/.config/satnogs-gut
 
 script:
-  - cd station && pytest -v && cd ..
-  - cd server && PYTHONTEST=. pytest -v
+  - cd station && pytest -v ; cd ..
+  - cd server && PYTHONTEST=. pytest -v ; cd ..
+  - shellcheck ${SHELL_SCRIPTS} ${SHELLCHECK_OPTS}
```
Comment (fivitti) [inline .travis.yml:37]: Could we use here the wildcard "*.sh"? Is it supported?

**Review by fivitti — CHANGES_REQUESTED** [review]
It is OK, but maybe we should add wildcard for shellcheck?

File: .travis.yml (hunk 1 above)
Comment (tomaszmrugalski) [reply .travis.yml:37]: Sure, we can. This will require fixing several scripts, though. Not a bit problem, though.

Comment (fivitti) [reply .travis.yml:37]: Ok, then I accept your MR and add an issue for improve shell scripts.

**Pull request merged**